src/components/MagPage/MagIcons.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import styles from './styles/MagIcons.css'
+
+let cx = classNames.bind(styles);
+
+class MagIcons extends React.Component {
```

Comment:
This is tiny, but maybe this class should be named MagIcon since it just renders one item.